pkg/porter/list_test.go

```diff
@@ -18,19 +18,19 @@ func TestNewDisplayInstallation(t *testing.T) {
 		cp := storage.NewTestInstallationProvider(t)
 		defer cp.Close()
 
-		i := cp.CreateInstallation(storage.NewInstallation("", "wordpress"), func(i *storage.Installation) {
+		install := cp.CreateInstallation(storage.NewInstallation("", "wordpress"), func(i *storage.Installation) {
 			i.Status.Action = cnab.ActionUpgrade
 			i.Status.ResultStatus = cnab.StatusRunning
 		})
 
-		i, err := cp.GetInstallation(context.Background(), "", "wordpress")
+		_, err := cp.GetInstallation(context.Background(), "", "wordpress")
 		require.NoError(t, err, "ReadInstallation failed")
 
-		di := NewDisplayInstallation(i)
+		di := NewDisplayInstallation(install)
```

Comment:
```suggestion
		displayinstall := NewDisplayInstallation(install)
```
I feel like if we make `i` less ambiguous we should follow the same with `di`  - but open to just leaving it because it is nbd

Reply:
This has been updated on the latest push.